src/index.js

```diff
@@ -127,8 +128,9 @@ const { WORKSPACE_EXTERNAL } = SLACK_TARGETS;
 export const main = wrap(run)
   .with(authWrapper, { authHandlers: [LegacyApiKeyHandler, ScopedApiKeyHandler, AdobeImsHandler] })
   .with(dataAccess)
-  .with(enrichPathInfo)
   .with(bodyData)
+  .with(multipartFormData)
+  .with(enrichPathInfo)
```

Comment:
@solaris007, I have moved `enrichPathInfo` to be "before" the `multipartFormData` (and `bodyData`) wrapper such that headers are accessible on `context.pathInfo`.